File: android/src/main/kotlin/uz/shs/video_player/models/ProgramsInfo.kt
package uz.shs.video_player.models

import com.google.gson.annotations.SerializedName
import java.io.Serializable

data class ProgramsInfo(
    @SerializedName("day")
    val day: String,
    @SerializedName("tvPrograms")
    val tvPrograms: List<TvProgram>,
) : Serializable

File: lib/models/programs_info.dart
import 'package:video_player/models/tv_program.dart';

class ProgramsInfo {
  const ProgramsInfo({required this.day, required this.tvPrograms});

  final String day;
  final List<TvProgram> tvPrograms;

  Map<String, dynamic> toMap() => {'day': day, 'tvPrograms': tvPrograms.map((v) => v.toMap()).toList()};

  @override
  String toString() => 'ProgramsInfo{day: $day, tvPrograms: $tvPrograms}';
}
